coding-the-legislation/10_basic_example.md

```diff
@@ -61,6 +61,54 @@ To make sure that the formula you have just written works the way you expect, yo
 
 You can check the [YAML tests documentation](writing_yaml_tests.md) to learn more about how to write YAML tests, and how to run them.
 
+## Example with Enum variable
+
+Let's say we have a `HOUSING_OCCUPANCY_STATUS` enumeration:
```

Comment:
This starts with an engineering need. A user does not start with “having an enumeration”, but with a modelling issue. Give a use case example, and explain why an `Enum` is the proper implementation before explaining how to do it in OpenFisca 😉